networking/using-rfhe.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY
+[id="using-rfhe"]
+= Monitoring Redfish hardware events
+include::_attributes/common-attributes.adoc[]
+:context: using-rfhe
+
+toc::[]
+
+:FeatureName: Redfish hardware events
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="installing-hw-events"]
+== Installing the Baremetal Hardware Event Proxy Operator
+
+Install the Baremetal Hardware Event Proxy Operator using the {product-title} CLI or the web console.
+
+include::modules/nw-rfhe-installing-operator-cli.adoc[leveloffset=+2]
+
+include::modules/nw-rfhe-installing-operator-web-console.adoc[leveloffset=+2]
+
+include::modules/hw-installing-amq-interconnect-messaging-bus.adoc[leveloffset=+1]
+
+[id="subscribing-hw-events"]
+== Subscribing to Redfish BMC hardware events for a cluster node
+
+As a cluster administrator, you can subscribe to Redfish BMC events generated on a node in your cluster by creating a `BMCEventSubscription` custom resource definition (CRD) for the node, and then creating a `HardwareEvent` CR for the event.
```

Comment:
`BMCEventSubscription` is a CR, not CRD.
FYI. This is the CRD https://github.com/metal3-io/baremetal-operator/blob/main/config/crd/bases/metal3.io_bmceventsubscriptions.yaml. The CRD is installed as part of operator, and it defines the syntax of a CR. Then users create CR to use the feature.

Reply:
As a cluster administrator, you can subscribe to Redfish BMC events generated on a node in your cluster by creating a `BMCEventSubscription` CR, then creating a `HardwareEvent` CR to process the received events.